components/cluster-as-a-service/staging/external-secrets.yaml

```diff
@@ -0,0 +1,56 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: cluster-as-a-service-hypershift-credentials
+  namespace: clusters
+spec:
+  dataFrom:
+  - extract:
+      key: staging/platform/terraform/it-cloud-aws-konflux-preprod-eaas01
+  - extract:
+      key: staging/eaas/konflux-eaas-stage
+  refreshInterval: 5m
+  secretStoreRef:
+    kind: ClusterSecretStore
+    name: appsre-stonesoup-vault
```

Comment:
Should this be `appsre-vault` instead? We seem to have both in use.

Reply:
I don't think that makes any difference.